hack/test-all-minikube-local-registry.sh

```diff
@@ -7,6 +7,30 @@ export KBLD_E2E_DOCKERHUB_HOSTNAME=$(minikube ip):30777
 # uncomment to disable stress tests
 # export KBLD_E2E_SKIP_STRESS_TESTS=true
 
-kapp deploy -a reg -f test/e2e/assets/minikube-local-registry.yml -y
+# Install secretgen to generate registry certs
+kapp deploy -a sg -f https://github.com/vmware-tanzu/carvel-secretgen-controller/releases/download/v0.8.0/release.yml -y
+# Install local docker2 registry
+kapp deploy -a reg -f <(ytt -f test/e2e/assets/minikube-local-registry.yml -v registry_alt_name=$(minikube ip)) -y
+
+if [ ! -z "$GITHUB_RUN_ID" ]; then
```

Comment:
Since the bash unbounded option is enabled above on line 2 `set -u` and most developers will *not* have `GITHUB_RUN_ID` env var set, running this script locally on a dev machine I think will fail.

Reply:
yup, i actually run into this myself... forgot to fix this.